src/lib/index.js

```diff
@@ -98,29 +99,46 @@ lib.pauseEvent = function(e) {
  * ------------------------------------------
  */
 
-// set VERBOSE to true to get a lot more logging and tracing
-lib.VERBOSE = false;
+/* eslint-disable no-console */
+lib.log = function() {
```

Comment:
I kinda like having them as a separate module if that's cool with you. Just means it's an extra `require` on most files.